### PR TITLE
RangeInput track color prop docs

### PR DESCRIPTION
**Pull request by g4rry420**
Signed-off-by: GurkiranSingh <gurkiransinghk@gmail.com>

Doc are added for the RangeInput track `color` prop.
https://github.com/grommet/grommet/pull/5275

### CODE REVIEW

File: src/screens/RangeInput.js [hunk 1]
```diff
@@ -64,6 +61,26 @@ export default () => (
...
         </Property>
 
+        <Property name="color">
+          <Description>A hex, name, or rgb value.</Description>
+          <PropertyValue type="string">
+            <Example defaultValue>"accent-1"</Example>
+          </PropertyValue>
+          <PropertyValue type="object">
+            <Description>For 'dark' and 'light' modes.</Description>
+            <Example>{`{"light": "string", "dark": "string"}`}</Example>
```
Comment (jcfilben) [inline src/screens/RangeInput.js:71]:
```suggestion
            <Description>An object with a color for dark and light modes.</Description>
            <Example>{`{ dark: "string", light: "string" }`}</Example>
```

File: src/screens/RangeInput.js [hunk 2]
```diff
@@ -64,6 +61,26 @@ export default () => (
...
+          </PropertyValue>
+          <PropertyValue type="array">
+            <Description>
+              An array of object with a color, value and opacity based on thumb
+              value.
+            </Description>
+            <Example>
+              {`[{ color: "string", value: number, opacity?: number }]`}
+            </Example>
```
Comment (jcfilben) [inline src/screens/RangeInput.js:80]:
```suggestion
            <Description>
              An array of objects with a color, value and opacity based on thumb
              value.
            </Description>
            <Example>
              {`[
  { value: 3, color: '#FF0000', opacity: 0.5 },
  { value: 7, color: { light: 'accent-3', dark: 'brand' } },
  { value: 10, color: '#00FF00' },
]`}
            </Example>
```

File: src/screens/RangeInput.js [hunk 3]
```diff
@@ -64,6 +61,26 @@ export default () => (
           <GenericA11yTitle />
         </Property>
 
+        <Property name="color">
+          <Description>A hex, name, or rgb value.</Description>
+          <PropertyValue type="string">
+            <Example defaultValue>"accent-1"</Example>
```
Comment (halocline) [inline src/screens/RangeInput.js:67]:
```suggestion
          <Description>The color of the track.</Description>
          <PropertyValue type="string">
            <Description>
              A hex, name, rgb value, or theme color name.
            </Description>
            <Example>"control"</Example>
```

**Comment by halocline** [comment]
Hah, @jcfilben -- Sorry, I had this open all day and made a bunch of comments overlapping with yours! Looks like we are on a similar train of thought. Apologies for the confusion.

**Review by jcfilben — APPROVED** [review]
Looks good to me!

**Review by halocline — APPROVED** [review]
Looks good, thank you @g4rry420! Much appreciated!

**Pull request merged**